pkg/agent/proxy/proxier.go

```diff
@@ -476,19 +494,21 @@ func (p *proxier) installServices() {
 					// of the Service are different, install two groups. One group has all Endpoints, the other has only
 					// local Endpoints.
 					groupID := p.groupCounter.AllocateIfNotExist(svcPortName, true)
-					if err = p.ofClient.InstallServiceGroup(groupID, affinityTimeout != 0, localEndpointUpdateList); err != nil {
+					// The Multi-cluster Service support ClusterIP type of Service only, so always set mcsLocalService to nil when
+					// it's NodePort type of Service.
+					if err = p.ofClient.InstallServiceGroup(groupID, affinityTimeout != 0, nil, localEndpointUpdateList); err != nil {
```

Comment:
Is it possible that the ClusterIP of a NodePort/LoadBalancer as the Endpoint of a mcService?